examples/http-certification/assets/src/backend/src/lib.rs

```diff
@@ -80,6 +58,11 @@ fn collect_assets<'content, 'path>(
 // Certification
 fn certify_all_assets() {
     // 1. Define the asset certification configurations.
+    let encodings = vec![
+        (AssetEncoding::Brotli, "br".to_string()),
+        (AssetEncoding::Gzip, "gzip".to_string()),
+    ];
```

Comment:
maybe we could have an impl of `AssetEncoding` that returns that tuple e.g. 

```suggestion
    let encodings = vec![AssetEncoding::config_brotli(), AssetEncoding::config_gzip()];
```

Reply:
Agreed, having some impl to avoid repetition would be good.
Maybe something like below would be better than a dedicated function for each encoding?
```
impl AssetEncoding {
  pub fn config_tuple(&self) -> (AssetEncoding, String) {
  ...
  }
}
```

Reply:
Thanks for the suggestion both.

I've added a slightly different approach like this:
```Rust
vec![AssetEncoding::Brotli.default(), AssetEncoding::Gzip.custom("gzip")]
```

What do you guys think?